### PR TITLE
Tagpicker Fix for overflowing long tags

### PR DESCRIPTION
**Pull request by gitjain**
#### Pull request checklist

- [ ] Addresses an existing issue: #1395
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: common/changes/tagpicker-overflowing_2017-04-04-22-16.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Tag picker: fix for overflowing long tags (Fix has been done but width of the Tag items needs to be fixed to see the ellipses in the drop down.)",
+      "type": "patch"
+    },
+    {
+      "comment": "",
+      "packageName": "@uifabric/utilities",
+      "type": "none"
```
Comment (joschect) [inline common/changes/tagpicker-overflowing_2017-04-04-22-16.json]: Remove one of the change files, there should only be 1.

File: packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss [hunk 2]
```diff
@@ -13,6 +13,11 @@
   white-space: nowrap;
   cursor: default;
   user-select: none;
+  display: flex;
+  flex-wrap: nowrap;
+  max-width: 100%;
+  position: absolute;
+  right: 2px;
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss]: I don't think this is quite right. you need at least top/left/right defined if you are going to position absolute it. Also, that doesn't make sense as it's not relative to anything (you can't default the root element to position absolute...

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss]: Yeah, there are some unnecessary styles here. I'm cleaning it up a bit. The position absolute is overriden elsewhere, so is unnecessary.

**Comment by micahgodbolt** [comment]
I removed a good chunk of unnecessary styles. As for the approach of using 60vw to limit the tag items, I don't have any problem with that (actually is probably better than limiting to width of input). Getting some feedback from the fabric design teams channel. 

image

In case of skinny input field in larger window.
image

File: packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss [hunk 3]
```diff
@@ -30,22 +27,24 @@
...
   font-size: $ms-font-size-s;
   display: inline-block;
   text-align: center;
   vertical-align: top;
   width: 30px;
   height: 100%;
+  flex-shrink: 0;
 }
 
+.tagItemTextOveflow {
```
Comment (mikewheaton) [inline packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss]: Typo here.

**Pull request merged**